index.src.html

```diff
@@ -1216,6 +1253,10 @@
     5. [=Queue a control message=] to encode |frameClone|.
 
     [=Running a control message=] to encode the frame means performing these steps.
+    1. If {{VideoEncoder/[[active color space]]}} is `null`:
+        1. If |frame|.{{VideoFrame/colorSpace}} is not `null`, assign it's value
+            to {{VideoEncoder/[[active color space]]}}.
+        2. ISSUE: what here? 709? Can we trust that frame.colorSpace is not null?
```

Comment:
@sandersdan thoughts? I we discussed it being null in some cases, like if you were creating from an internal videoframe that had some non-compatible representation? I wasn't able to quickly find cases in our impl where this actually occurs.

Reply:
That's correct, an implementation could produce a VideoFrame with no JS-exposed color space. In Chrome that can happen easily with the current implementation because the WebCodecs implementation only supports a subset of color spaces (BT. 601, BT. 709, and sRGB), but if/when it's expanded to all of H.273 it would only happen for frames that are in a non-H.273 color space (currently not possible in Chrome).